src/components/LandingPageBlocks/HeroBlock/animationScripts/animationCodeText.js

```diff
@@ -0,0 +1,24 @@
+export let code_first = 
```

Comment:
Why do we have separate variables for every element? In case you need to introduce a new element, you would have to change a lot of things. Elements should be stored in an array of objects. Something like:

```javascript
const codeSnippets = [
    {
        title: "Microservices",
        snippet: "code_snippet_goes_here"
    },
    ...
]
```

And your code should be just take this array as an input.